### PR TITLE
added creation of Test Case test

### PR DESCRIPTION
**Pull request by rocketby**
**Preconditions**
- User is logged in Allure testOps
- Project with specified name already exists

**Actions**
1. Open project with specified name
2. From Sidebar navigate to Test Cases Table
3. Unlock test cases menu tree
4. Create new test case with unique name

**Asserts**
- Check, that number of test cases increased by 1 in test cases table
- Check, that test case with specified unique name is displayed in tese cases table 

5. Delete the created test case:
  5.1. Choose the test case
  5.2. Click bulk operations button
  5.3. Choose delete option
  5.4. Confirm deletion of the test case

### CODE REVIEW

File: src/test/java/cloud/autotests/tests/TestCasesTests.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import cloud.autotests.pages.TestCasesTable;
+import io.qameta.allure.Story;
+import org.junit.jupiter.api.Test;
+import java.sql.Timestamp;
+import static com.codeborne.selenide.Selenide.open;
+import static io.qameta.allure.Allure.step;
+
+@Story("Test cases tests")
+public class TestCasesTests extends TestBase {
+    Timestamp timestamp = new Timestamp(System.currentTimeMillis());
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/TestCasesTests.java:17]: унести в RandomUtils или Faker

File: src/test/java/cloud/autotests/tests/TestCasesTests.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    private static final String PROJECT_NAME = "teacher qa_guru_diplom_project";
+    private String testCaseName = "Test0001_Tat_"+timestamp.getTime(); //generate unique name of TC each time
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/TestCasesTests.java:19]: пробелы до и после +

File: src/test/java/cloud/autotests/tests/TestCasesTests.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    void shouldCreateTestCaseTest() {
+        TestCasesTable testCasesTable = new TestCasesTable();
+
+        step("Navigating to Test Cases menu", () -> {
+            ProjectsTable projectsTable = open(App.config.webUrl(), ProjectsTable.class);
+            ProjectPage projectPage = projectsTable.navigateTo(PROJECT_NAME);
+            projectPage.getSidebar().navigateTo(MenuItem.TEST_CASES);
+        });
+
+        step("Unlock test cases menu tree", () -> {
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/TestCasesTests.java:32]: дублируется степ - в тесте и в PO

File: src/test/java/cloud/autotests/tests/TestCasesTests.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            testCasesTable.unlockTestCasesTree();
+        });
+
+        step("Create new test case and check, that number of test cases increased", () -> {
+            int numberOfTestCases = testCasesTable.getNumberOfTestCases();
+            testCasesTable.createNewTestCase(testCaseName);
+            testCasesTable.shouldHaveSize(numberOfTestCases + 1);
+        });
+
+        step("Check, that new test case is displayed in test cases table", () -> {
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/TestCasesTests.java:42]: убрать степы, тоже

**Review by svasenkov — CHANGES_REQUESTED** [review]

File: src/test/java/cloud/autotests/tests/TestCasesTests.java (hunk 1 above)
Comment (rocketby) [reply src/test/java/cloud/autotests/tests/TestCasesTests.java]: fixed

File: src/test/java/cloud/autotests/tests/TestCasesTests.java (hunk 2 above)
Comment (rocketby) [reply src/test/java/cloud/autotests/tests/TestCasesTests.java]: fixed

File: src/test/java/cloud/autotests/tests/TestCasesTests.java (hunk 3 above)
Comment (rocketby) [reply src/test/java/cloud/autotests/tests/TestCasesTests.java]: fixed

File: src/test/java/cloud/autotests/tests/TestCasesTests.java (hunk 4 above)
Comment (rocketby) [reply src/test/java/cloud/autotests/tests/TestCasesTests.java]: fixed

**Pull request merged**